roles/openshift_master/defaults/main.yml

```diff
@@ -19,3 +19,6 @@ r_openshift_master_os_firewall_allow:
 - service: etcd embedded
   port: 4001/tcp
   cond: "{{ groups.oo_etcd_to_config | default([]) | length == 0 }}"
+
+registry_auth_credentials_path: "/var/lib/{{ openshift.common.service_type }}/.docker"
```

Comment:
`registry_auth_credentials_path: "{{ openshift.common.data_dir }}/.docker"` instead? This should be pretty stable but that's the var we use elsewhere.